airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -55,6 +58,7 @@ class RedshiftDataOperator(BaseOperator):
 
     template_fields = (
         "cluster_identifier",
+        "workgroup_name",
```

Comment:
not sure whether this will need to be reordered

Reply:
Tests are passing but i'll do it to remain consistent.